### DISCUSSION TITLE
Trending repositories shows same repository multiple times

### DISCUSSION THREAD

**Discussion by biosfood** [community/community#82720] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When looking at the trending page, I noticed that some repositories where shown multiple times (up to 5).
The bug seems to be mostly visual but the HTML sent to the browser already contains the cloned entries.
Starring and unstarring one of the copies does not update the others, they only get updated when refreshing the page. Navigating to repositories works as expected.
The section "trending users" seems to not be effected
Browser: firefox
Here is a screenshot (the duplicated top bar is not noticable when viewing the page and just comes from the screenshot):
![trending-bug](https://github.com/community/community/assets/64695423/72193ed5-de49-4df6-bd03-d8aa0d4add58)

**Comment by vipunsanjana** [comment]
your internet speed check
